tools/ktest-tool/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+#===------------------------------------------------------------------------===#
+#
+#                     The KLEE Symbolic Virtual Machine
+#
+# This file is distributed under the University of Illinois Open Source
+# License. See LICENSE.TXT for details.
+#
+#===------------------------------------------------------------------------===#
+install(PROGRAMS ktest-tool DESTINATION bin)
+
+# Copy into the build directory's binary directory
+# so integration tests can find it
+configure_file(ktest-tool "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/ktest-tool" COPYONLY)
```

Comment:
Developers should just read the CMake documentation it's pretty clear that `install()` and `configure_file()` are different.

`install()` is for installing files when the `make install` target is invoked. The `configure_file()` line is to copy the file into the `bin/` directory in the build tree such that if the original source file (`ktest-tool` here) is modified it triggers CMake to automatically re-configure itself and copy the modified file over.